about.html

```diff
@@ -20,45 +23,49 @@
     </div>
   </header>
   <h1>About</h1>
-  <h3>Room: 307</h3>
-  <h3>Time: Mondays - Thursdays, immediately after 10th period - 5:00pm</h3>
-  <h2>What is the dojo?</h2>
+  <h4>Room: 307</h4>
+  <h4>Time: Mondays - Thursdays, immediately after 10th period - 5:00pm</h4>
+  <h3>What is the dojo?</h3>
   <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
     tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
     quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
     consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
     cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
     non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-  <h2>Rules</h2>
-  <ol start = 0>
-    <li>All lists must be 0-indexed.</li>
-    <li>No food or drink in the dojo (H20 okay).</li>
-    <li>Be respectful to your peers.</li>
-    <li>Clean up after yourself.</li>
-    <li>Don't make excessively loud noises.</li>
-    <li>Avoid using illegal internet (Facebook/Messenger).</li>
-    <li>Keep the aisles clear of backpacks.</li>
-    <li>Lock your workstation when not in use (CTRL + ALT + L).</li>
+<h3>Rules</h3>
+
+  <ol start = 0 class="mdl-list">
+    <li class="mdl-list__item">All lists must be 0-indexed.</li>
+    <li class="mdl-list__item">No food or drink in the dojo (H20 okay).</li>
+    <li class="mdl-list__item">Be respectful to your peers.</li>
+    <li class="mdl-list__item">Clean up after yourself.</li>
+    <li class="mdl-list__item">Don't make excessively loud noises.</li>
+    <li class="mdl-list__item">Avoid using illegal internet (Facebook/Messenger).</li>
+    <li class="mdl-list__item">Keep the aisles clear of backpacks.</li>
+    <li class="mdl-list__item">Lock your workstation when not in use (CTRL + ALT + L).</li>
   </ol>
-  <h2>Sensei and Senpai</h2>
+
+  <h3>Sensei and Senpai</h3>
   <img src = "sensei.jpg"/>
   <p>From left to right: ...</p>
-  <h3>Sensei: Brian Moses, Emma Choi, Greg Zborovsky, and William Cao</h3>
-  <h3>Senpai</h3>
+  <h4>Sensei: Brian Moses, Emma Choi, Greg Zborovsky, and William Cao</h4>
+  <h4>Senpai</h4>
+  <div class="mdl-grid">
   <ul>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
-    <li>Lorem Ipsum</li>
+    <li class="mdl-list__item">Lorem Ipsum</li>
```

Comment:
Can we organize the list of senpai in a year book style. Checkout this for help https://getmdl.io/components/index.html#layout-section/grid